src/main/services/team/provisioning/__tests__/TeamProvisioningDiagnosticsHelpers.test.ts

```
@@ -0,0 +1,59 @@
+import type { TeamProvisioningSupportDiagnostic } from '@shared/types';
+import { describe, expect, it } from 'vitest';
+
+import {
+  buildProvisioningTraceDetail,
+  pushUniqueSupportDiagnostics,
+} from '../TeamProvisioningDiagnosticsHelpers';
+
+describe('TeamProvisioningDiagnosticsHelpers', () => {
+  describe('buildProvisioningTraceDetail', () => {
+    it('returns undefined when no extras are present', () => {
+      expect(buildProvisioningTraceDetail(undefined)).toBeUndefined();
```

ESLint

[error] Remove this redundant "undefined".

(sonarjs/no-undefined-argument)